From e84821a62568e5eb49c3b72d63c1b77d74692159 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Tue, 8 Jul 2025 11:57:58 +0200
Subject: [PATCH] Use exprtk from Yocto

Upstream-Status: Inappropriate [embedded specific]
---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ef98373c..e43aa437 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,7 +34,6 @@ endif(SUPPRESS_AUTO_DOC_BUILD)
 
 FIND_PACKAGE(Boost COMPONENTS thread system chrono filesystem REQUIRED)
 FIND_PACKAGE(ChimeraTK-cppext 01.05 REQUIRED)
-FIND_PACKAGE(exprtk 01.04 REQUIRED)
 
 FIND_PACKAGE(nlohmann_json 03.11 REQUIRED)
 add_compile_definitions(JSON_DIAGNOSTICS=1)
@@ -114,7 +113,7 @@ target_compile_options(${PROJECT_NAME} PUBLIC ${projectCxxFlags})
 target_include_directories(${PROJECT_NAME} INTERFACE "$<INSTALL_INTERFACE:include>")
 set_target_properties(${PROJECT_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_FULL_LIBRARY_VERSION} SOVERSION ${${PROJECT_NAME}_SOVERSION})
 target_link_libraries(${PROJECT_NAME}
-  PRIVATE ${CMAKE_DL_LIBS} PkgConfig::LibXML++ ChimeraTK::exprtk
+  PRIVATE ${CMAKE_DL_LIBS} PkgConfig::LibXML++
 
   # we set this public because of Backend libs using shm_open (e.g. test code)
   PUBLIC ${RT_LIBRARIES}
