From 272cef1ae3cc732396e1aa397e3725aa334b270b Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Fri, 15 Mar 2024 15:23:09 +0100
Subject: [PATCH] Add missing <memory> include

---
 backends/uio/include/UioAccess.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/backends/uio/include/UioAccess.h b/backends/uio/include/UioAccess.h
index 60ca0ff4..0e3596aa 100644
--- a/backends/uio/include/UioAccess.h
+++ b/backends/uio/include/UioAccess.h
@@ -5,6 +5,7 @@
 #include <boost/filesystem.hpp>
 
 #include <atomic>
+#include <memory>
 #include <string>
 
 namespace ChimeraTK {
