From 3d1944b3127cc9cd4db5e54631cfe7ede9bf4f1c Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Tue, 19 Mar 2024 16:51:32 +0100
Subject: [PATCH] Add missing include for cstdint

---
 device_backends/mmio/include/MmioAccess.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/device_backends/mmio/include/MmioAccess.h b/device_backends/mmio/include/MmioAccess.h
index 06a44f31..80a2feb2 100644
--- a/device_backends/mmio/include/MmioAccess.h
+++ b/device_backends/mmio/include/MmioAccess.h
@@ -4,6 +4,7 @@
 
 #include "DeviceFile.h"
 
+#include <cstdint>
 #include <string>
 
 namespace ChimeraTK {
