From 81fb85dbe2032abea530f7ddf4d43676106e45f7 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Tue, 11 Apr 2023 16:35:11 +0200
Subject: [PATCH] Bump LibXML++ version

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5d771aba..a17365b3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,7 +37,7 @@ FIND_PACKAGE(ChimeraTK-cppext 01.05 REQUIRED)
 add_dependency(exprtk 01.00 REQUIRED)
 
 FIND_PACKAGE(PkgConfig REQUIRED)
-set(LIBXML++_VERSION "libxml++-2.6")
+set(LIBXML++_VERSION "libxml++-5.0")
 PKG_CHECK_MODULES(LibXML++ REQUIRED IMPORTED_TARGET ${LIBXML++_VERSION})
 PKG_CHECK_MODULES(glib REQUIRED IMPORTED_TARGET glib-2.0)
 
