From b423a61c8f19740340356ada0800a9bafe9d0002 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Fri, 15 Sep 2023 12:47:50 +0200
Subject: [PATCH] Drop debug message

---
 device_backends/uio/src/UioAccess.cc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/device_backends/uio/src/UioAccess.cc b/device_backends/uio/src/UioAccess.cc
index b8f6b1d6..9a84024b 100644
--- a/device_backends/uio/src/UioAccess.cc
+++ b/device_backends/uio/src/UioAccess.cc
@@ -66,9 +66,6 @@ namespace ChimeraTK {
       throw ChimeraTK::logic_error("UIO: Multiple memory regions are not supported");
     }
 
-    std::cout << "Calling uio-access write " << map << " " << address << ":" << (void*)data << " " << sizeInBytes
-              << std::endl;
-
     // This is a temporary work around, because register nodes of current map use absolute bus addresses.
     //NOLINTNEXTLINE(cppcoreguidelines-pro-type-reinterpret-cast)
     address = address % reinterpret_cast<uint64_t>(_deviceKernelBase);
