diff --git a/CMakeLists.txt b/CMakeLists.txt
index 54fce739..152dc8ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,7 +33,6 @@ add_dependency(exprtk REQUIRED 01.00)
 FIND_PACKAGE(PkgConfig REQUIRED)
 set(LIBXML++_VERSION "libxml++-5.0")
 PKG_CHECK_MODULES(LibXML++ REQUIRED ${LIBXML++_VERSION})
-PKG_CHECK_MODULES(glib REQUIRED glib-2.0)
 include_directories(${glib_INCLUDE_DIRS} ${LibXML++_INCLUDE_DIRS})
 
 # The PCIe backend can only be built on Linux, so we define a variable here that
