From 90ee8912dc0c59b85eb1402f0430e81ab8ca5862 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 19 Apr 2023 13:40:47 +0200
Subject: [PATCH] Install XML Generator

Upstream-Status: Inappropriate [enable feature]
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c2a868f..5f9485e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,4 +85,4 @@ enable_testing()
 add_test(testGenericServer ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/testGenericServer)
 
 #install executables
-install(TARGETS ${EXE_NAME} RUNTIME DESTINATION bin)
+install(TARGETS ${EXE_NAME} ${EXE_NAME}-xmlGenerator RUNTIME DESTINATION bin)
