From 19802962ba7c8c5e4e6bd36d19695fe08fb7560f Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 12 Apr 2023 13:40:28 +0200
Subject: [PATCH] Drop RPATH from python bindings

---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 942b427..4f90def 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -177,11 +177,6 @@ set_tests_properties(testPushType PROPERTIES TIMEOUT 5)
 # enable coverage in debug Mode(CMAKE_BUILD_TYPE == "Debug").
 include(enable_code_coverage_report)
 
-# do not remove runtime path to libmtca-deviceaccess location  from
-# ${boost_python_core_module}  when installing
-set_property(TARGET ${boost_python_core_module}
-  PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
-
 get_python_module_install_path(PYTHONLIBS_VERSION_STRING install_path)
 message(STATUS "Install Path: ${install_path}")
 
