From fda13d0f89a2a1b9a443940eaac17400759038da Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Tue, 17 Feb 2026 12:39:49 +0100
Subject: [PATCH] Disable stubgen

Upstream-Status: Inappropriate [embedded specific]
---
 CMakeLists.txt | 25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 248782b..9e725bc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,12 +23,6 @@ find_package(ChimeraTK-DeviceAccess 03.23 REQUIRED)
 set(PYBIND11_FINDPYTHON ON)
 find_package(pybind11 2.10)
 
-find_program(STUBGEN stubgen)
-
-if(${STUBGEN} STREQUAL "STUBGEN-NOTFOUND")
-  message(FATAL_ERROR "Could not find program 'stubgen'. Try installing package 'mypy' or similar.")
-endif()
-
 # ==============================================================================#
 
 # Define library for the C++ part
@@ -51,21 +45,6 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE
 # Copy Python part to build directory (required for tests to run)
 file( COPY mtca4u.py DESTINATION ${PROJECT_BINARY_DIR} )
 
-# ==============================================================================#
-# Python module stub
-if((NOT ${CMAKE_BUILD_TYPE} STREQUAL "asan") AND(NOT ${CMAKE_BUILD_TYPE} STREQUAL "tsan"))
-  configure_file(cmake/runStubgen.sh.in ${CMAKE_CURRENT_BINARY_DIR}/runStubgen.sh)
-  add_custom_target(${PROJECT_NAME}_interface ALL
-    COMMAND ./runStubgen.sh
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-    DEPENDS ${PROJECT_NAME}
-    BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pyi
-    COMMENT "Use stubgen to create .pyi for statement completion")
-else()
-  # stubgen would fail to run due to missing asan/tsan symbols
-  message(NOTICE "Note: Python stub will not be generated for asan and tsan builds.")
-endif()
-
 # ==============================================================================#
 ENABLE_TESTING()
 
@@ -125,10 +104,6 @@ else()
 endif()
 set (matlab_helper_path "share/ChimeraTK-DeviceAccess-PythonBindings-${${PROJECT_NAME}_MAJOR_VERSION}.${${PROJECT_NAME}_MINOR_VERSION}")
 
-if((NOT ${CMAKE_BUILD_TYPE} STREQUAL "asan") AND(NOT ${CMAKE_BUILD_TYPE} STREQUAL "tsan"))
-  # stubgen does not work for asan/tsan builds (see above), hence we skip installation of the interface file
-  install( FILES ${PROJECT_BINARY_DIR}/deviceaccess.pyi DESTINATION ${install_path} )
-endif()
 install( FILES ${PROJECT_SOURCE_DIR}/mtca4u.py DESTINATION ${install_path} )
 install( TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${install_path} )
 install(FILES ${PROJECT_SOURCE_DIR}/pybind11_builtins.py DESTINATION ${matlab_helper_path})
