From d12360906ff50729ef4c1580ceda8b1a4fe96d3b Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 12 Apr 2023 14:02:41 +0200
Subject: [PATCH] Bump libxml++ version to 5.0

---
 CMakeLists.txt               | 2 +-
 src/PersistentDataStorage.cc | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 85b0683..59db6c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,7 +47,7 @@ FIND_PACKAGE(ChimeraTK-DeviceAccess 03.14 REQUIRED)
 
 # libXML++ is used for parsing/writing XML files
 FIND_PACKAGE(PkgConfig REQUIRED)
-set(LIBXML++_VERSION "libxml++-2.6")
+set(LIBXML++_VERSION "libxml++-5.0")
 PKG_CHECK_MODULES(LibXML++ REQUIRED IMPORTED_TARGET ${LIBXML++_VERSION})
 
 file(GLOB_RECURSE library_headers ${CMAKE_SOURCE_DIR}/include/ChimeraTK/*.h
diff --git a/src/PersistentDataStorage.cc b/src/PersistentDataStorage.cc
index 4c05e29..cb28ccf 100644
--- a/src/PersistentDataStorage.cc
+++ b/src/PersistentDataStorage.cc
@@ -64,7 +64,7 @@ namespace ChimeraTK {
         if(!_variableRegisteredFromApp[i]) continue; // exclude variables no longer present in the application
 
         // create XML element for the variable and set name attribute
-        xmlpp::Element* variable = rootElement->add_child("variable");
+        xmlpp::Element* variable = rootElement->add_child_element("variable");
         variable->set_attribute("name", static_cast<std::string>(_variableNames[i]));
 
         // generate value XML tags and set type name as a string
@@ -104,7 +104,7 @@ namespace ChimeraTK {
     }
     // add one child element per element of the value
     for(size_t idx = 0; idx < pValue->size(); ++idx) {
-      xmlpp::Element* valueElement = parent->add_child("val");
+      xmlpp::Element* valueElement = parent->add_child_element("val");
       valueElement->set_attribute("i", userTypeToUserType<std::string>(idx));
       valueElement->set_attribute("v", userTypeToUserType<std::string>((*pValue)[idx]));
     }
