From 164a199ad69df08563296025fc854758ab35c778 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 12 Apr 2023 14:18:32 +0200
Subject: [PATCH] Remove explicit glib dependency


---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e55cf43..ff094db 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,6 @@ FIND_PACKAGE(ChimeraTK-DeviceAccess 03.09 REQUIRED)
 FIND_PACKAGE(PkgConfig REQUIRED)
 set(LIBXML++_VERSION "libxml++-2.6")
 PKG_CHECK_MODULES(LibXML++ REQUIRED IMPORTED_TARGET ${LIBXML++_VERSION})
-PKG_CHECK_MODULES(glib REQUIRED IMPORTED_TARGET glib-2.0)
 
 # Find BOOST filesystem
 # Note: we need to search this before looking for the unit_test_framework, since we must not link against the
@@ -67,7 +66,7 @@ target_link_libraries(${PROJECT_NAME}
     # put ControlSystemAdapter public since implicit dep DeviceAccess must be public
     PUBLIC ChimeraTK::ChimeraTK-ControlSystemAdapter
     PUBLIC ${Boost_LIBRARIES}
-    PRIVATE Threads::Threads PkgConfig::LibXML++ PkgConfig::glib ${HDF5_LIBRARIES})
+    PRIVATE Threads::Threads PkgConfig::LibXML++ ${HDF5_LIBRARIES})
 
 # do not remove runtime path of the library when installing
 set_property(TARGET ${PROJECT_NAME} PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
