From d04af48fb3fa9c2d4003fe5971ce1120154ee343 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 12 Apr 2023 14:35:32 +0200
Subject: [PATCH 3/3] Remote install rpath

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c613595..d56e6e7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,9 +68,6 @@ target_link_libraries(${PROJECT_NAME}
     PUBLIC ${Boost_LIBRARIES}
     PRIVATE Threads::Threads PkgConfig::LibXML++ ${HDF5_LIBRARIES})
 
-# do not remove runtime path of the library when installing
-set_property(TARGET ${PROJECT_NAME} PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
-
 # add a target to generate API documentation with Doxygen
 include(cmake/enable_doxygen_documentation.cmake)
 
-- 
2.40.0

