From db59548b0d48f32336b46b72d3f36be8f5107595 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Thu, 26 Jun 2025 16:34:24 +0000
Subject: [PATCH] Drop unneccessary constructor. Fails to compile with gcc9

---
 Python/bindings/include/PyDataConsistencyGroup.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Python/bindings/include/PyDataConsistencyGroup.h b/Python/bindings/include/PyDataConsistencyGroup.h
index 3e56d0c..816e094 100644
--- a/Python/bindings/include/PyDataConsistencyGroup.h
+++ b/Python/bindings/include/PyDataConsistencyGroup.h
@@ -39,7 +39,6 @@ namespace ChimeraTK {
    protected:
     // Helper constructor that takes a std::vector and passes it on to the iterator constructor of
     // ReadAnyGroup
-    PyDataConsistencyGroup(std::vector<TransferElementAbstractor> args) : _impl(args.begin(), args.end()) {}
 
     DataConsistencyGroup _impl;
   };
