From a59048993ca78681d0ea95d3ab8be1888956a847 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 12 Apr 2023 14:35:32 +0200
Subject: [PATCH] Remote install rpath

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9b2ce66..81f862f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,9 +72,6 @@ target_link_libraries(${PROJECT_NAME}
   PUBLIC ${Boost_LIBRARIES}
   PRIVATE Threads::Threads PkgConfig::LibXML++ PkgConfig::glib ${HDF5_LIBRARIES})
 
-# do not remove runtime path of the library when installing
-set_property(TARGET ${PROJECT_NAME} PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
-
 # add a target to generate API documentation with Doxygen
 include(cmake/enable_doxygen_documentation.cmake)
 
