From ea988d51d8086f9ecd13889ed124645605418083 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 12 Apr 2023 14:35:32 +0200
Subject: [PATCH] Remote install rpath

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c5858d9..25d5b2a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,9 +75,6 @@ target_link_libraries(${PROJECT_NAME}
 
 target_compile_definitions(${PROJECT_NAME} PRIVATE CHIMERATK_INSIDE_APPLICATION_CORE)
 
-# do not remove runtime path of the library when installing
-set_property(TARGET ${PROJECT_NAME} PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
-
 # add a target to generate API documentation with Doxygen
 include(cmake/enable_doxygen_documentation.cmake)
 
