From 711e02bd88b11832e094321dbbe66801936f48a6 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Tue, 25 Jul 2023 15:52:29 +0200
Subject: [PATCH] Drop glib2 dependency

---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 81f862f..0bfb241 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,6 @@ FIND_PACKAGE(ChimeraTK-DeviceAccess 03.11 REQUIRED)
 FIND_PACKAGE(PkgConfig REQUIRED)
 set(LIBXML++_VERSION "libxml++-5.0")
 PKG_CHECK_MODULES(LibXML++ REQUIRED IMPORTED_TARGET ${LIBXML++_VERSION})
-PKG_CHECK_MODULES(glib REQUIRED IMPORTED_TARGET glib-2.0)
 
 # Find BOOST filesystem
 # Note: we need to search this before looking for the unit_test_framework, since we must not link against the
@@ -70,7 +69,7 @@ target_link_libraries(${PROJECT_NAME}
   # put ControlSystemAdapter public since implicit dep DeviceAccess must be public
   PUBLIC ChimeraTK::ChimeraTK-ControlSystemAdapter
   PUBLIC ${Boost_LIBRARIES}
-  PRIVATE Threads::Threads PkgConfig::LibXML++ PkgConfig::glib ${HDF5_LIBRARIES})
+  PRIVATE Threads::Threads PkgConfig::LibXML++ ${HDF5_LIBRARIES})
 
 # add a target to generate API documentation with Doxygen
 include(cmake/enable_doxygen_documentation.cmake)
