From f5c41d8912288f68965a621e5027846ea7d00923 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 19 Apr 2023 15:41:48 +0200
Subject: [PATCH] Remove link flags

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3f7706a..c12d0df 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,7 +37,7 @@ include_directories(${PROJECT_BINARY_DIR}/include)
 set(EXEC_NAME mtca4u_fw_programmer)
 add_executable(${EXEC_NAME} ${app_sources} )
 set_target_properties(${EXEC_NAME} PROPERTIES COMPILE_FLAGS "${CMAKE_CXX_FLAGS}")
-set_target_properties(${EXEC_NAME} PROPERTIES LINK_FLAGS "${ChimeraTK-DeviceAccess_LINK_FLAGS} ${Boost_LINK_FLAGS} ${CURSES_LINK_FLAGS}")
+set_target_properties(${EXEC_NAME} PROPERTIES LINK_FLAGS "${Boost_LINK_FLAGS} ${CURSES_LINK_FLAGS}")
 target_link_libraries(${EXEC_NAME} ${ChimeraTK-DeviceAccess_LIBRARIES} ${Boost_LIBRARIES} ${CURSES_LIBRARIES})
 
 #change the install prefix to the source directory in case the user has not specified a destination
