From 66c923dbce08ae5aa3cf9094fd1d4591b2a30d12 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 19 Apr 2023 13:40:47 +0200
Subject: [PATCH] Install XML Generator

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dea5aa7..7d683fa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,4 +83,4 @@ enable_testing()
 add_test(testGenericServer ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/testGenericServer)
 
 #install executables
-install(TARGETS ${EXE_NAME} RUNTIME DESTINATION bin)
+install(TARGETS ${EXE_NAME} ${EXE_NAME}-xmlGenerator RUNTIME DESTINATION bin)
