From 95a759a3cb765dbd878340fb089c0e1005db0a41 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Wed, 19 Apr 2023 13:24:35 +0200
Subject: [PATCH] Do not leak system paths into build

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed0c72c..20d3250 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,14 +56,12 @@ add_library(${PROJECT_BASE_NAME} STATIC ${library_sources} )
 # main server executable
 add_executable(${PROJECT_NAME} ${server_sources})
 set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "${ChimeraTK-ApplicationCore_CXX_FLAGS}")
-set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "${ChimeraTK-ApplicationCore_LINK_FLAGS} ${Adapter_LINK_FLAGS}")
 target_link_libraries(${PROJECT_NAME} ${PROJECT_BASE_NAME} ${ChimeraTK-ApplicationCore_LIBRARIES}
                       ${Adapter_LIBRARIES}
                       ${ChimeraTK-DeviceAccess-DoocsBackend_LIBRARIES})
 
 add_executable(${PROJECT_NAME}-xmlGenerator ${server_sources})
 set_target_properties(${PROJECT_NAME}-xmlGenerator PROPERTIES COMPILE_FLAGS "-DGENERATE_XML")
-set_target_properties(${PROJECT_NAME}-xmlGenerator PROPERTIES LINK_FLAGS "${ChimeraTK-ApplicationCore_LINK_FLAGS} ${Adapter_LINK_FLAGS}")
 target_link_libraries(${PROJECT_NAME}-xmlGenerator ${PROJECT_BASE_NAME} ${ChimeraTK-ApplicationCore_LIBRARIES}
                       ${ChimeraTK-DeviceAccess-DoocsBackend_LIBRARIES})
 
@@ -85,7 +83,6 @@ include_directories(${Boost_INCLUDE_DIRS})
 foreach(target writeWordStatus testGenericServer)
     add_executable(${target} tests/${target}.cc)
     set_target_properties(${target} PROPERTIES COMPILE_FLAGS "${ChimeraTK-ApplicationCore_CXX_FLAGS}")
-    set_target_properties(${target} PROPERTIES LINK_FLAGS "${ChimeraTK-ApplicationCore_LINK_FLAGS}")
     target_link_libraries(${target} ${PROJECT_BASE_NAME} ${ChimeraTK-ApplicationCore_LIBRARIES} ${Boost_LIBRARIES})
 endforeach()
 
