From c9fafe2036142d0d87ee63e19aff1f53847e6d1a Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Thu, 9 Feb 2023 12:55:29 +0100
Subject: [PATCH] Drop hard dependency on DOOCS

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3506129..f2f374e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,7 +35,6 @@ include(${CMAKE_SOURCE_DIR}/cmake/add_dependency.cmake)
 include(cmake/enable_code_coverage_report.cmake)
 include(cmake/set_control_system_adapter.cmake)
 add_dependency(ChimeraTK-ApplicationCore 02.02 REQUIRED)
-add_dependency(ChimeraTK-DeviceAccess-DoocsBackend REQUIRED)
 
 #FIND_PACKAGE(Boost COMPONENTS unit_test_framework REQUIRED)
 
