From 9d2ff9fab21bcd5f84ac701f909aa6bad2d162b7 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Fri, 22 Jul 2022 15:58:45 +0200
Subject: [PATCH] Hard-code numpy include paths

Upstream-Status: Inappropriate [embedded specific]
---
 src/tools/python.jam | 20 +++-----------------
 1 file changed, 3 insertions(+), 17 deletions(-)

diff --git a/src/tools/python.jam b/src/tools/python.jam
index 0dfc750a2..39b2f10ce 100644
--- a/src/tools/python.jam
+++ b/src/tools/python.jam
@@ -847,23 +847,9 @@ local rule configure ( version ? : cmd-or-prefix ? : includes * : libraries ? :
     #
     # Discover the presence of NumPy
     #
-    debug-message "Checking for NumPy..." ;
-    local full-cmd = "import sys; sys.stderr = sys.stdout; import numpy; print(numpy.get_include())" ;
-    local full-cmd = $(interpreter-cmd)" -c \"$(full-cmd)\"" ;
-    debug-message "running command '$(full-cmd)'" ;
-    local result = [ SHELL $(full-cmd) : strip-eol : exit-status ] ;
-    if $(result[2]) = 0
-    {
-        .numpy = true ;
-        .numpy-include = $(result[1]) ;
-        debug-message "NumPy enabled" ;
-    }
-    else
-    {
-        debug-message "NumPy disabled. Reason:" ;
-        debug-message "  $(full-cmd) aborted with " ;
-        debug-message "  $(result[1])" ;
-    }
+    # Just hardcode numpy paths
+    .numpy = true ;
+    .numpy-include = $(libraries)/site-packages/numpy/_core/include ;
 
     #
     # End autoconfiguration sequence.
