From 0131aba67358e57370df0f5748caf64024159fb3 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georgd@desy.de>
Date: Fri, 25 Apr 2025 15:25:10 +0000
Subject: [PATCH] fix: Make it possible to override KERNEL_SRC

---
 Makefile.kernel | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/Makefile.kernel b/Makefile.kernel
index bb31715..f952471 100755
--- a/Makefile.kernel
+++ b/Makefile.kernel
@@ -2,11 +2,15 @@ mtcadummy-objs := mtcadummy_drv.o mtcadummy_proc.o mtcadummy_firmware.o mtcadumm
 obj-m += mtcadummy.o
 
 KVERSION = $(shell uname -r)
+KERNEL_SRC = /lib/modules/$(KVERSION)/build
 
 ccflags-y = -Wall -Wuninitialized
 
 all:
-	make -C /lib/modules/$(KVERSION)/build V=1 M=$(PWD) modules
+	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) modules
+
+modules_install:
+	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) modules_install
 
 #Compile with debug flag, causes lots of kernel output, which fills the logs
 #and slows down the driver.
@@ -19,7 +23,7 @@ full_debug:
 	KCPPFLAGS="-DPCIEUNI_DEBUG -fprofile-arcs -ftest-coverage" make all
 
 clean:
-	make -C /lib/modules/$(KVERSION)/build V=1 M=$(PWD) clean
+	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) clean
 
 #the two possible locations where the gpcieuni include files can be
 #/usr/local in case of a local, manual installation by the admin
