From 998736f08ceb812aa1673a3be2dcc7a509a42c40 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Tue, 6 May 2025 11:14:16 +0200
Subject: [PATCH] Remove old configuration files

Upstream-Status: Submitted [https://github.com/ChimeraTK/project-template]
---
 cmake/create_cmake_config_files.cmake | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/cmake/create_cmake_config_files.cmake b/cmake/create_cmake_config_files.cmake
index 8143f8f..8386c37 100644
--- a/cmake/create_cmake_config_files.cmake
+++ b/cmake/create_cmake_config_files.cmake
@@ -301,13 +301,6 @@ endif()
 configure_file(cmake/PROJECT_NAMEConfigVersion.cmake.in.in "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}ConfigVersion.cmake.in" @ONLY)
 configure_file(${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}ConfigVersion.cmake.in "${PROJECT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake" @ONLY)
 
-# create the pkgconfig file
-configure_file(cmake/PROJECT_NAME.pc.in.in "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}.pc.in" @ONLY)
-configure_file(${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}.pc.in "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.pc" @ONLY)
-
-# install configuration file for pkgconfig
-install(FILES "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.pc" DESTINATION share/pkgconfig COMPONENT dev)
-
 if(${PROVIDES_EXPORTED_TARGETS})
     # generate and install export file
     install(EXPORT ${PROJECT_NAME}Targets
